packages/app/src/components/menu/titlebar/Titlebar.jsx

```diff
@@ -21,20 +21,18 @@ const Toolbar = styled(ButtonToolbar)`
 // TODO move title to separate componen
 // TODO solve the font for the whole app
 
-export default class Titlebar extends React.Component {
-  render() {
-    return <Toolbar>
-      <ButtonGroup>
-        <ShowPanelsButton content={<FiMenu/>} tip="Menu"/>
-        <TitleButton content={<FiChevronRight/>} tip="Terminal"></TitleButton>
-      </ButtonGroup>
-
-      <span data-field="title" style={{ margin: '10px' }}>3D Molecular Viewer</span>
-
-      <ButtonGroup>
-        <TitleButton content={<AiOutlinePicture/>} tip="Display mode"/>
-        <TitleButton content={<GiDrop/>} tip="Display color"></TitleButton>
-      </ButtonGroup>
-   </Toolbar>;
-  }
-}
+const Titlebar = ({ loadingStage }) => <Toolbar>
```

Comment:
Please forgive me for the lack of React knowledge 😄 I would prefer the classic `function` declaration instead of arrow functions here. Does it go against the newest trend? The arrow functions are great when used inline (as other functions' arguments) but here we just define a named function actually.

Let's use `function` here and in other similar places if you don't mind.

Reply:
Oh, my. Maybe, I've chosen a wrong line to comment on :) This one is a data-only function, and it looks clearer as an arrow function...